tests/third_party/cupy/core_tests/test_ndarray_copy_and_view.py

```diff
@@ -274,9 +274,10 @@ def test_fill(self, xp, dtype):
         a.fill(1)
         return a
 
-    @testing.with_requires("numpy>=1.24.0")
-    @testing.for_all_dtypes_combination(("dtype1", "dtype2"))
-    @testing.numpy_cupy_array_equal(accept_error=ComplexWarning)
+    @pytest.mark.skip(
+        "asynchronous fill does not allow numpy scalar array for consistency"
+        "with dpnp.fill_diagonal"
+    )
     def test_fill_with_numpy_scalar_ndarray(self, xp, dtype1, dtype2):
```

Comment:
Could you please update `test_fill_with_nonscalar_ndarray` test to remove skip there? (need to accept both exception type: TypeError and ValueError)